source/extensions/filters/http/admission_control/evaluators/default_evaluator.cc

```diff
@@ -0,0 +1,68 @@
+#include "extensions/filters/http/admission_control/evaluators/default_evaluator.h"
+
+#include <algorithm>
+
+#include "envoy/grpc/status.h"
+
+#include "common/common/enum_to_int.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AdmissionControl {
+
+DefaultResponseEvaluator::DefaultResponseEvaluator(
+    envoy::extensions::filters::http::admission_control::v3alpha::AdmissionControl::
+        DefaultEvaluationCriteria evaluation_criteria) {
+  // HTTP status.
+  if (evaluation_criteria.http_status_size() > 0) {
+    for (const auto& range : evaluation_criteria.http_status()) {
+      http_success_fns_.emplace_back([range](uint64_t status) {
+        return (static_cast<uint64_t>(range.start()) <= status) &&
+               (status < static_cast<uint64_t>(range.end()));
+      });
+    }
+  } else {
+    // We default to all 5xx codes as request failures.
+    http_success_fns_.emplace_back([](uint64_t status) { return status < 500; });
```

Comment:
these two lines together got me real confused since the comment talks about what we consider request failures, then the next line defines the success condition instead. Maybe reword the comment?